### ISSUE TITLE
Focus issue on CreateFullPage with modal

### ISSUE THREAD

**Issue by hkang92** [carbon-design-system/ibm-cloud-cognitive#2702]
## What package(s) are you using?

- [x] Carbon for IBM Products (or Carbon for IBM Cloud & Cognitive)
- [ ] Legacy/CDAI
- [ ] Legacy/Security

## Detailed description

> Describe in detail the issue you're having.

 I am using the CreateFullPage component and had a question with accessibility. I clicked on the cancel button on the FullPage and it opened up a modal and when I press tab, it focuses on the enabled next button on the outside before going into the modal. Is there a way to shift focus on the modal?

It is also seen here https://carbon-for-ibm-products.netlify.app/?path=/story/ibm-products-patterns-create-flows-createfullpage--create-full-page

> Is this issue related to a specific component?

CreateFullPage

> What did you expect to happen? What happened instead? What would you like to
> see changed?

The focus should be on the modal instead of the outside

> What browser are you working in?

Chrome and FireFox

> What version of the @carbon/ibm-products (or @carbon/ibm-cloud-cognitive)
> package are you using?

    "@carbon/ibm-products": "^1.43.0",

## Additional information

![ab38ce90-194f-4dbb-858d-8bbee84ec1a9](https://user-images.githubusercontent.com/38839544/221245781-f5143e5d-0907-46bb-add2-6469f8695b2c.png)
![Screen Recording 2023-02-24 at 11 22 10 AM](https://user-images.githubusercontent.com/38839544/221246210-e40db65b-6213-47b2-a883-1cac00339c54.gif)

**Comment by elycheea** [comment]
@hkang92 this was addressed in #2790 which should be available in our next release!
